### PR TITLE
WIP: Add clarification for gregorian date and time values

### PR DESCRIPTION
**Pull request by spectranaut**

### CODE REVIEW

File: spec/datetimeformat.html [hunk 1]
```diff
@@ -364,14 +364,22 @@ <h1>FormatDateTimeToParts ( _dateTimeFormat_, _x_ )</h1>
...
+        1. Apply calendrical calculations on _t_ for the given _calendar_ and _timeZone_ to produce weekday, era, year, month, day, hour, minute, second, and inDST values. The calculations should use best available information about the specified _calendar_ and _timeZone_, including current and historical information about time zone offsets from UTC and daylight saving time rules.
+        1. Note: If the _calendar_ is `"gregory"`, then the calculations of weekday, year, month, day, hour, minute and second must match the algorithms specified in ES2020, <emu-xref href="#sec-overview-of-date-objects-and-definitions-of-abstract-operations"></emu-xref>. Specifically:
```
Comment (leobalter) [inline spec/datetimeformat.html]: Remove `Note: `?

File: spec/datetimeformat.html [hunk 2]
```diff
@@ -364,14 +364,22 @@ <h1>FormatDateTimeToParts ( _dateTimeFormat_, _x_ )</h1>
...
+          1. The value for weekday must be the value `WeekDay(t)` specific in ES2020, <emu-xref href="#sec-week-day"></emu-xref>
```
Comment (leobalter) [inline spec/datetimeformat.html]: the wording is odd, maybe something like `Let value for weekday be `WeekDay(t)`...` ?

Comment (leobalter) [reply spec/datetimeformat.html]: I also need to render this to understand how this href gets

**Comment by leobalter** [comment]
I pushed a suggestion commit, feel free to revert or remote it anytime.

**Comment by spectranaut** [comment]
Add an assert:

1. Assert:_t_ is a number value -- something about time clip.

**Comment by leobalter** [comment]
maybe: "_t_ is time clipped number value"

**Pull request closed without merging**